loaders/src/glTF/2.0/babylon.glTFLoader.ts

```diff
@@ -670,16 +674,18 @@ module BABYLON.GLTF2 {
         babylonMesh.computeWorldMatrix(true);
 
         // Set morph target manager after all vertices data has been processed
-        babylonMesh.morphTargetManager = morphTargetManager;
+        if (morphTargetManager !== undefined && (<MorphTargetManager>morphTargetManager).numInfluencers != 0) {

```

Comment:
change the declaration from `var morphTargetManager` to `var morphTargetManager: MorphTargetManager` so that you don't need this cast.